web/handlers/admin/handler_test.go

```diff
@@ -19,9 +19,12 @@ func TestDashoard(t *testing.T) {
 	html, resp := ts.Client.GetHTML(url.String())
 	a.Equal(http.StatusOK, resp.Code, "wrong HTTP status code")
 
+	a.Equal(1, html.Find("#online").Size())
```

Comment:
was about to suggest this should but `.Length()` but TIL:

> Size(), which is an alias for Length()


You could test the actual content if you add the `Count()` func and use `ts.Members.CountReturns(23, nil)` to have it display 23.